scripts/globals/treasure.lua

```diff
@@ -1107,7 +1107,7 @@ xi.treasure.treasureInfo =
                 {
                     {
                         test = function(player)
-                            return player:getCharVar('WildCard') == 2
+                            return player:getCharVar('WildCard') == 2 or player:getCharVar('WildCard') == 3
```

Comment:
This variable is never set to 2 in the first place. Its just used in this file.

Variable being checked should be `'[2][77]Prog'`

So the fix would be in the end...
```lua
    misc =
    {
        {
            test = function(player)
                return (player:getCharVar('WildCard') == 2 or player:getCharVar('WildCard') == 2) and not player:hasKeyItem(xi.ki.JOKER_CARD)
            end,

            code = function(player)
                npcUtil.giveKeyItem(player, xi.ki.JOKER_CARD)
            end,
        },
    },
```

Reply:
That makes a lot more sense (in regards to the char var). Appreciate the suggestion, refactors have been made and pushed.

Reply:
For future cases, please dont resolve conversations yourself. Wait for the original reviewer to close it, sowe dont miss stuff on accident.